tests/Views/Frontend/CatalogViewTest.php

```diff
@@ -10,17 +10,11 @@ class CatalogViewTest extends TestCase
 {
     public function testGetIndex()
     {
-        $itemRepositoryMock = $this->createMock(ItemRepository::class);
-        $itemRepositoryMock->expects($this->any())
-            ->method('listValues')
-            ->willReturn(collect());
-        $itemRepositoryMock->expects($this->once())
-            ->method('search')
-            ->willReturn(new SearchResult(collect(), 0));
-        $this->app->instance(ItemRepository::class, $itemRepositoryMock);
-
         $response = $this->get('/katalog');
+
         $response->assertStatus(200);
+        $response->assertViewIs('frontend.catalog-new.index');
```

Comment:
This might be a good opportunity to re-name the "new" catalog to just `catalog`? :)
```suggestion
        $response->assertViewIs('frontend.catalog.index');
```